media_player/alexa.py

```diff
@@ -979,6 +992,33 @@ def _get_request(self, uri, data=None):
                 message))
             return None
 
+    def get_last_device_serial(self):
+        """Identify the last device's serial number."""
+        try:
+            response = self._get_request('/api/activities?startTime=&size=1&offset=1')
+        except Exception as ex:
+            template = ("An exception of type {0} occurred."
+                        " Arguments:\n{1!r}")
+            message = template.format(type(ex).__name__, ex.args)
+            _LOGGER.error("An error occured accessing the API: {}".format(
+                message))
+            return None
+
+        try:
+            last_activity = response.json()['activities'][0]
```

Comment:
@brianhanifin Not sure if you saw my comment as you closed the convo, but let's combine this with the prior try block so we're not duplicating the catch.